src/server/api/routers/openmediavault.ts

```diff
@@ -0,0 +1,168 @@
+import axios from 'axios';
+import Consola from 'consola';
+import { z } from 'zod';
+import { checkIntegrationsType, findAppProperty } from '~/tools/client/app-properties';
+import { getConfig } from '~/tools/config/getConfig';
+
+import { createTRPCRouter, protectedProcedure } from '../trpc';
+
+let sessionId: any = '';
+let loginToken: any = '';
+
+export const openmediavaultRouter = createTRPCRouter({
+  auth: protectedProcedure
```

Comment:
anything that is used to display information that you (or more specifically another user) potentially want anonymous visitors to see, needs to be publicProcedure.
Any authentication is done in the backend and credential can't be passed on to the front end anyway, so there's no worry there.